content/docs/details/showing-dependencies/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Showing Dependencies Between Components
+publishedDate: '2022-06-13T14:49:47.0Z'
+description: How to model dependencies between components
+---
+
+You can find a thorough guide to modelling relationships or dependencies between components in Roadie Backstage in this [blog post](https://roadie.io/blog/modelling-software-backstage/).
```

Comment:
Not a big deal but AFAIK f you leave out the `https://roadie.io` it will load the page like a single page application rather than a full page refresh.